about/index.md

```diff
@@ -5,151 +5,238 @@ layout: info
 
 See also:
 
--   [18F Blog: Facts about Publishing Open Source Code](https://18f.gsa.gov/2016/08/08/facts-about-publishing-open-source-code-in-government/)
--   [The entire Pro Git book, written by S](https://git-scm.com/book/en/v2)
+- [18F Blog: Facts about Publishing Open Source Code][gsa-18f]
+- [The entire Pro Git book, written by S][git-scm]
 
-Contributing to NIST Open Source Projects
-=========================================
+## Contributing to NIST Open Source Projects
 
-NIST welcomes contributions from the general public to our [open source projects on GitHub](https://github.com/usnistgov). 
+NIST welcomes contributions from the general public to our
+[open source projects on GitHub][usnistgov].
 
 ### Contributing
 
-Refer to individual projects for their requirements on accepting contributions. In general though, we follow the "fork and pull" Git workflow model:
-
--   Fork a repository
--   Develop your changes in your fork
--   Create a pull request to the "upstream" repository
--   If approved, changes will be merged in by a repository maintainer
+Refer to individual projects for their requirements on accepting
+contributions. In general though, we follow the "fork and pull" Git
+workflow model:
+
+- Fork a repository
+- Develop your changes in your fork
+- Create a pull request to the "upstream" repository
+- If approved, changes will be merged in by a repository maintainer.
+
+<!-- links -->
+
+[code-gov]: https://code.gov/
+[code-json]: https://code.nist.gov/explore/code.json
+[fed-code]: https://code.gov/agency-compliance/compliance/dashboard
+[gh-18f]: https://github.com/18f/pages-server
+[gh-eml]: https://github.com/settings/emails
+[gh-help]: https://help.github.com/articles/about-topics/
+[gh-link]: https://help.github.com/articles/adding-an-email-address-to-your-github-account/
+[gh-pages]: https://github.com/usnistgov/pages-root/wiki
+[gh-ppl]: https://github.com/usnistgov/people
+[gh-prof]: https://help.github.com/categories/setting-up-and-managing-your-github-profile/
+[gh-sign]: https://github.com/usnistgov/discuss/issues/2
+[git-id]: https://git-scm.com/book/en/v2/Getting-Started-First-Time-Git-Setup#Your-Identity
+[git-scm]: https://git-scm.com/book/en/v2
+[gsa-18f]: https://18f.gsa.gov/2016/08/08/facts-about-publishing-open-source-code-in-government/
+[llnl-code]: https://software.llnl.gov/
+[nist-code]: https://code.nist.gov
+[nist-open]: https://www.nist.gov/open/copyright-fair-use-and-licensing-statements-srd-data-software-and-technical-series-publications
+[nist-ppl]: https://www.nist.gov/about-nist/our-organization/people
+[nist-pubs]: https://inet.nist.gov/adlp/publishing-instructions/publishing-software
+[nist-soft]: https://www.nist.gov/open/copyright-fair-use-and-licensing-statements-srd-data-software-and-technical-series-publications#software
+[nist-tax]: https://data.nist.gov/od/id/691DDF3315711C14E0532457068146BE1907
+[open-lic]: https://github.com/usnistgov/opensource-repo/blob/main/LICENSE.md
```

Comment:
Probably should be a relative link?

Reply:
No, this repo ([opensource](/usnistgov/opensource)) is LLNL's work, distributed under the MIT license. Repos under [usnistgov](/usnistgov) should have the NIST disclaimer of copyright & warranty, the canonical version of which is maintained (by this team) under the template [opensource-repo](/usnistgov/opensource-repo).

Reply:
Thanks for the feeback guys!
Thanks @tkphd